staking_deposit/intl/en/cli/existing_mnemonic.json

```diff
@@ -8,7 +8,7 @@
         },
         "arg_mnemonic": {
             "help": "The mnemonic that you used to generate your keys. (It is recommended not to use this argument, and wait for the CLI to ask you for your mnemonic as otherwise it will appear in your shell history.)",
-            "prompt": "Please enter your mnemonic separated by spaces (\" \")"
+            "prompt": "Please enter your mnemonic separated by spaces (\" \"). Note: you only need to enter the first 4 letters of each word if you'd prefer."
```

Comment:
- What do you think about translation in the future release?
- 4-letters abbreviations are not available in Chinese and Korean so we may want to avoid adding translation in these languages.

Reply:
As you say, Chinese, Japanese & Korean don't have abbreviated versions. And I'm a little concerned about collisions between 4-letter mnemonics in the romance languages. I've done some testing here, but I'm by no means convinced it can't happen (and trigger the sanity assert) especially considering all the alternate forms of letters (eg moving accents etc).

As scuh, I think we should avoid translating this bit at all. ie. if you use not-English (tm) you will have to type out the full mnemonic. At a later point if we implement interactive predictive text for words, we can address this.